src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -49,6 +49,18 @@ public boolean isValidViewer() {
     public boolean isTeam() {
         return this == TEAMS || this == OWN_TEAM;
     }
+    
+    public boolean isOwnTeamMembers() {
+        return this == OWN_TEAM_MEMBERS;
+    }
+    
+    public boolean isOtherStudentsInTheCourse() {
+        return this == STUDENTS;
```

Comment:
For this PR, this is unnecessary here too, just do this check directly at its call site.